src/app/auth/auth.service.ts

```diff
@@ -1,10 +1,9 @@
 import { Injectable } from '@angular/core';
 import { AngularFireAuth } from '@angular/fire/auth';
 import { auth } from 'firebase/app';
-import { AuthModule } from './auth.module';
 
 @Injectable({
-  providedIn: AuthModule
+  providedIn: 'root'
```

Comment:
@ipriel had to change this because it introduced a circular dependency...

Reply:
OK. It should be fine either way.